task1.py

```diff
@@ -0,0 +1,14 @@
+def search_word(element, my_list):
+    if element in my_list:
+        return True
+    else:
+        return False
+
+
+print("Enter The Value")
+value = input()
+my_list1 = ['Lahore', 'Pakistan', 'Karachi', 'Peshawar', 'Quetta']
```

Comment:
Read the words from file and and then add them in list.

Reply:
ok